insights/parsers/journalctl.py

```diff
@@ -0,0 +1,154 @@
+"""
+JournalCtl - command ``journalctl xxx``
+=======================================
+
+This module contains the following parsers:
+
+JournalAll - file ``/sos_commands/logs/journalctl_--no-pager``
+--------------------------------------------------------------
+JournalctlHeader - command ``/usr/bin/journalctl --header``
+-----------------------------------------------------------
+JournalSinceBoot - file ``/sos_commands/logs/journalctl_--no-pager_--boot``
+---------------------------------------------------------------------------
+
+"""
+
+
+from insights.core.plugins import parser
+from insights.core import CommandParser, LegacyItemAccess
+from insights.parsers import SkipException, get_active_lines
+from insights.specs import Specs
+from .. import Syslog
```

Comment:
Better change this line to `from insights.core import Syslog`, to use the same style in the same module.